### ISSUE TITLE
Incorrect I202 flag raised

### ISSUE THREAD

**Issue by adithyabsk** [terrencepreilly/darglint#20]
If the following code has a docstring

```python
def check_module_installed(name):
    """Temp

    Args:
        name (str): module name

    Returns:
        bool: Whether the module can be imported

    """
    try:
        __import__(name)
    except ImportError:
        return False
    else:
        return True
```

This snipped raises `I202 Excess "Returns" in Docstring: + return`

**Comment by terrencepreilly** [comment]
Apparently, the `body` attribute of the ast node representing try-statements doesn't include the `except` or `else` branches.  So when walking over the tree, I was missing that.  I won't be surprised if this happens elsewhere in the AST.  For now, though, at least this case should be handled.

Resolved in v0.5.5.
